src/sentry/receivers/onboarding.py

```diff
@@ -195,20 +192,16 @@ def record_first_transaction(project, event):
     )
 
     try:
-        user = Organization.objects.get(id=project.organization_id).get_default_owner()
+        default_user_id = project.organization.get_default_owner().id
     except IndexError:
-        logging.getLogger("sentry").warn(
-            "Cannot record first transaction for organization (%s) due to missing owners",
-            project.organization_id,
-        )
-        return
+        default_user_id = None
 
     analytics.record(
         "first_transaction.sent",
-        user_id=user.id,
+        default_user_id=default_user_id,
```

Comment:
I changed this to align with https://github.com/getsentry/sentry/blob/master/src/sentry/receivers/transactions.py#L18-L24 so that we send the same data to analytics as previously. This particular code path would never have been previously triggered as it's impossible for `record_first_event()` which calls this to ever get called with a transaction.